styles/variables-silicon.less

```diff
@@ -182,6 +182,10 @@
 @agate-item-label-font-size:  30px;
 @agate-item-side-outset:      30px;
 @agate-item-sub-text-size:    24px;
+@agate-item-itemcontent-font-size:    33px;
```

Comment:
Do we need defined this variable?
Is it not enough to define it already in agate-item-sizing(); ?

Reply:
Oh, ok. I deleted this variable.